htdocs/compta/bank/class/account.class.php

```diff
@@ -54,14 +55,6 @@ class Account extends CommonObject
     var $clos;
     var $rappro=1;    // If bank need to be conciliated
     var $url;
-    //! BBAN field for French Code banque
-    var $code_banque;
```

Comment:
What about old data? In spain we use CCC: Bank code, Branch code, control code and Account number. We officially use IBAN but people still use CCC nowadays